liferay-search/liferay-portal
# [TS] LPS-175924 Add missing Builder field

Motivation
=======
https://issues.liferay.com/browse/LPS-175924
https://issues.liferay.com/browse/LPS-164848 introduced the setAssetVocabularyLocalService method to the AssetCategoriesSearchFacetDisplayContextBuilder class, but did not call it in the Classic Search Widget.

Proposed Solution
========
Call the new method to set the field of the Builder

Steps to Verify
========
1. Start the bundle and log in as admin
2. Go to Categorization > Categories and create a new vocabulary with a new category
3. Go to Content & Data > Web content and create a new web content named "web"
4. Set the created category to the web content
5. Configure Web Search to use the Classic widget: Control Panel > System Settings > Search > Web Search > Classic Search Widget in Front Page (turn it on), save the changes
6. Go to the home page and search for "web"

**Expected behaviour:** Search results are rendered with categories facets
**Actual behaviour:** Portlet is not available and the log show a NullPointerException
